<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotWindow</class>
 <widget class="QMainWindow" name="BotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telegram Bot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/Small Icon.png</normaloff>:/Icons/Small Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="botPlainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBot">
    <property name="title">
     <string>Bot</string>
    </property>
    <addaction name="actionGetUpdate"/>
    <addaction name="actionSendMessage"/>
    <addaction name="actionGetMe"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBot"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGetUpdate"/>
   <addaction name="actionSendMessage"/>
   <addaction name="actionGetMe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGetUpdate">
   <property name="text">
    <string>Get Update</string>
   </property>
  </action>
  <action name="actionSendMessage">
   <property name="text">
    <string>Send Message</string>
   </property>
  </action>
  <action name="actionGetMe">
   <property name="text">
    <string>Get Me</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRestoreWindow">
   <property name="text">
    <string>Restore Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
